SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -44,13 +42,20 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, co
 	// Accept new player.
 	URLString.Append(TEXT("?workerAttribute=")).Append(UTF8_TO_TCHAR(CallerAttribute));
```

Comment:
May be marginally better to first convert `CallerAttribute` into an `FString` to use here as well as for `WorkersWithPlayersSpawned` below.

Reply:
Sure. Will tidy up the function now that the formats are the same.